### DISCUSSION TITLE
Does 0.14B2 support JSMPEG still?

### DISCUSSION THREAD

**Discussion by coryschak** [blakeblackshear/frigate#11804] [General Support]
### Describe the problem you are having

The previous build had a drop down to select MSE, JSMPEG and WEBRTC but with the new 0.14 i don't see that option anywhere.
Is it possible to change the stream to that again or provide the option?

### Version

0.14B2

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Frigate stats

```json
NA
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

HIkvision

**Comment by NickM-27** [comment]
yes, the UI will switch to jsmpeg in certain circumstances like when low bandwidth is detected.

**Reply by coryschak** [reply]
Is there a way to force it? The reason i ask is JSMPEG works in my tesla when driving and i want to be able to force that stream via a URL or drop down when viewing in my car.
